src/Library/demos/Color Dialog/main.blp

```diff
@@ -0,0 +1,51 @@
+using Gtk 4.0;
+using Adw 1;
+
+Adw.StatusPage {
+  title: "Color Dialog";
+  description: _("Show a dialog to select a color");
+
+  Box {
+    orientation: vertical;
+    halign: center;
+    spacing:42;
+
+    Box{
+      orientation:vertical;
+
+      ColorDialogButton color_dialog_button {
+        halign: center;
+        margin-bottom:12;
+      }
+      Label {
+        label: _("Color Dialog Button");
```

Comment:
```suggestion
        label: _("Dialog Button");
```

I'd also move these labels above their widgets and add something like the `title-2` class, so its clearer two different things are being demonstrated.

Reply:
What do you think of using a ``title-4`` instead of a ``title-2``?

![image](https://user-images.githubusercontent.com/58233418/226251626-5d847586-26f2-40e9-b3f7-b418fa5a7275.png)

![image](https://user-images.githubusercontent.com/58233418/226251757-75fd1be5-df66-4f44-970e-edc2ccfa241b.png)

Reply:
I think a small widget under a very big title looks a little off?

Reply:
Yeah, that does look better :+1: